corehq/apps/app_manager/static/app_manager/js/multimedia_size_util.js

```diff
@@ -12,13 +12,14 @@ hqDefine('app_manager/js/multimedia_size_util',[
     };
     var multimediaSizesView = function (url) {
         var self = {};
+        self.url = url;
```

Comment:
This should be an observable, and then you can set up a `subscribe` in here to call `load` when the URL changes, instead of making the callers do that work.

Reply:
https://github.com/dimagi/commcare-hq/pull/26032/commits/1c30039682ec1158ce087335c8d6e95dd6bb9928